js/widgets/help.js

```diff
@@ -48,7 +48,11 @@ function HelpWidget() {
         this.widgetWindow.getWidgetBody().append(this._helpDiv);
 
         // Make help div apeear in center of screen
-        this.widgetWindow.sendToCenter();
```

Comment:
I think we need to do this differently. sendToCenter() itself should check to see if width/height are set yet. If not, then we should use some reasonable set of numbers, maybe 150, 100. (Not sure where you came up with 404, 449.)

Reply:
Exactly. I was also proposing the same that we modify `sendToCenter()` to incorporate this. 

Actually when we use `sendToCenter()` function on help div, the (x, y) parameters set are (404, 449), so I hardcoded them. You can notice that the position of help div is the same what it would have been if we've used `sendToCenter()`

Reply:
I will adjust the parameters so that the mouse and starting blocks are not being shadowed.